examples/patterns/forms/forms.html

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: Forms / Layout
+category: _patterns
+---
+
+<form class="p-form p-form--stacked u-float-left">
```

Comment:
Ah see I thought the same but you can also just do a forms without classes so this doesn't really apply. Also there are quite a few discussions about when you should change form layout or not. Best to keep it as is for the time being.